<!doctype html><html lang=en-us>
<head>
<meta charset=utf-8>
<meta name=viewport content="width=device-width,initial-scale=1">
<meta http-equiv=x-ua-compatible content="IE=edge">
<meta name=generator content="Wowchemy 5.4.0 for Hugo">
<link rel=preconnect href=https://fonts.gstatic.com crossorigin>
<link rel=preload as=style href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap">
<link rel=stylesheet href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap" media=print onload="this.media='all'">
<meta name=description content="There are two main types of root systems in flowering plants, namely taproot systems of dicots and fibrous root systems found in monocots. Despite this fundamental split, our current knowledge of cellular and molecular mechanism driving root development is mainly based on studies of the dicot model Arabidopsis. However, the world major crops are monocots and little is known about the transcriptional programs underlying cell-type specification in this clade. Here, we report the transcriptomes of more than 20 000 single cells derived from root tips of two agronomically important rice cultivars. Using combined computational and experimental analyses we were able to robustly identify most of the major cell types and define novel cell-type-specific marker genes for both cultivars. Importantly, we found divergent cell types associated with specific regulatory programs, including phytohormone biosynthesis, signaling, and response, which were well conserved between the two rice cultivars. In addition, we detected substantial differences between the cell-type transcript profiles of Arabidopsis and rice. These species-specific features emphasize the importance of analyzing tissues across diverse model species, including rice. Taken together, our study provides insight into the transcriptomic landscape of major cell types of rice root tip at single-cell resolution and opens new avenues to study cell-type specification, function, and evolution in plants.
">
<link rel=alternate hreflang=en-us href=https://erc-decode.github.io/decode-website-academic/publication/liu-2021/>
<meta name=theme-color content="#F39200">
<link rel=stylesheet href=../../css/vendor-bundle.min.f1ecf783c14edc00c9320c205831ad8e.css media=print onload="this.media='all'">
<link rel=stylesheet href=https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.2.1/build/styles/github.min.css crossorigin=anonymous title=hl-light media=print onload="this.media='all'">
<link rel=stylesheet href=https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.2.1/build/styles/dracula.min.css crossorigin=anonymous title=hl-dark media=print onload="this.media='all'" disabled>
<link rel=stylesheet href=https://cdn.jsdelivr.net/npm/leaflet@1.7.1/dist/leaflet.min.css integrity crossorigin=anonymous media=print onload="this.media='all'">
<link rel=stylesheet href=../../css/wowchemy.2180088b006fdb53f44584d468183fbb.css>
<link rel=manifest href=../../manifest.webmanifest>
<link rel=icon type=image/png href=../../media/icon_hu84ca8f59585900c7228465808651375d_2429_32x32_fill_lanczos_center_3.png>
<link rel=apple-touch-icon type=image/png href=../../media/icon_hu84ca8f59585900c7228465808651375d_2429_180x180_fill_lanczos_center_3.png>
<link rel=canonical href=https://erc-decode.github.io/decode-website-academic/publication/liu-2021/>
<meta property="twitter:card" content="summary_large_image">
<meta property="og:site_name" content="ERC DECODE">
<meta property="og:url" content="https://erc-decode.github.io/decode-website-academic/publication/liu-2021/">
<meta property="og:title" content="Transcriptional landscape of rice roots at the single-cell resolution | ERC DECODE">
<meta property="og:description" content="There are two main types of root systems in flowering plants, namely taproot systems of dicots and fibrous root systems found in monocots. Despite this fundamental split, our current knowledge of cellular and molecular mechanism driving root development is mainly based on studies of the dicot model Arabidopsis. However, the world major crops are monocots and little is known about the transcriptional programs underlying cell-type specification in this clade. Here, we report the transcriptomes of more than 20 000 single cells derived from root tips of two agronomically important rice cultivars. Using combined computational and experimental analyses we were able to robustly identify most of the major cell types and define novel cell-type-specific marker genes for both cultivars. Importantly, we found divergent cell types associated with specific regulatory programs, including phytohormone biosynthesis, signaling, and response, which were well conserved between the two rice cultivars. In addition, we detected substantial differences between the cell-type transcript profiles of Arabidopsis and rice. These species-specific features emphasize the importance of analyzing tissues across diverse model species, including rice. Taken together, our study provides insight into the transcriptomic landscape of major cell types of rice root tip at single-cell resolution and opens new avenues to study cell-type specification, function, and evolution in plants.
"><meta property="og:image" content="https://erc-decode.github.io/decode-website-academic/publication/liu-2021/featured.jpg">
<meta property="twitter:image" content="https://erc-decode.github.io/decode-website-academic/publication/liu-2021/featured.jpg"><meta property="og:locale" content="en-us">
<meta property="article:published_time" content="2021-03-01T00:00:00+00:00">
<meta property="article:modified_time" content="2021-03-01T00:00:00+00:00">
<script type=application/ld+json>{"@context":"https://schema.org","@type":"Article","mainEntityOfPage":{"@type":"WebPage","@id":"https://erc-decode.github.io/decode-website-academic/publication/liu-2021/"},"headline":"Transcriptional landscape of rice roots at the single-cell resolution","image":["https://erc-decode.github.io/decode-website-academic/publication/liu-2021/featured.jpg"],"datePublished":"2021-03-01T00:00:00Z","dateModified":"2021-03-01T00:00:00Z","author":{"@type":"Person","name":"Qing Liu"},"publisher":{"@type":"Organization","name":"ERC DECODE","logo":{"@type":"ImageObject","url":"https://erc-decode.github.io/decode-website-academic/media/logo_hu755fd628a59dfbfb6631c31ab0c8a3a3_17790_192x192_fit_lanczos_3.png"}},"description":"There are two main types of root systems in flowering plants, namely taproot systems of dicots and fibrous root systems found in monocots. Despite this fundamental split, our current knowledge of cellular and molecular mechanism driving root development is mainly based on studies of the dicot model Arabidopsis. However, the world major crops are monocots and little is known about the transcriptional programs underlying cell-type specification in this clade. Here, we report the transcriptomes of more than 20 000 single cells derived from root tips of two agronomically important rice cultivars. Using combined computational and experimental analyses we were able to robustly identify most of the major cell types and define novel cell-type-specific marker genes for both cultivars. Importantly, we found divergent cell types associated with specific regulatory programs, including phytohormone biosynthesis, signaling, and response, which were well conserved between the two rice cultivars. In addition, we detected substantial differences between the cell-type transcript profiles of Arabidopsis and rice. These species-specific features emphasize the importance of analyzing tissues across diverse model species, including rice. Taken together, our study provides insight into the transcriptomic landscape of major cell types of rice root tip at single-cell resolution and opens new avenues to study cell-type specification, function, and evolution in plants.\n"}</script>
<script src=https://cdn.jsdelivr.net/gh/osano/cookieconsent@3.1.1/build/cookieconsent.min.js integrity="sha512-yXXqOFjdjHNH1GND+1EO0jbvvebABpzGKD66djnUfiKlYME5HGMUJHoCaeE4D5PTG2YsSJf6dwqyUUvQvS0vaA==" crossorigin=anonymous></script>
<link rel=stylesheet href=https://cdn.jsdelivr.net/gh/osano/cookieconsent@3.1.1/build/cookieconsent.min.css integrity="sha512-LQ97camar/lOliT/MqjcQs5kWgy6Qz/cCRzzRzUCfv0fotsCTC9ZHXaPQmJV8Xu/PVALfJZ7BDezl5lW3/qBxg==" crossorigin=anonymous>
<script>window.addEventListener("load",function(){window.cookieconsent.initialise({palette:{popup:{background:"#F39200",text:"rgb(255, 255, 255)"},button:{background:"rgb(255, 255, 255)",text:"#F39200"}},theme:"classic",content:{message:"This website uses cookies to ensure you get the best experience on our website.",dismiss:"Got it!",link:"Learn more",href:"/privacy/"}})})</script>
<title>Transcriptional landscape of rice roots at the single-cell resolution | ERC DECODE</title>
</head>
<body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class=page-wrapper data-wc-page-id=fe1966bee3de6c0517059633acd10cca>
<script src=../../js/wowchemy-init.min.8f76bdc9e086322ed5147724ebba3d06.js></script>
<aside class=search-modal id=search>
<div class=container>
<section class=search-header>
<div class="row no-gutters justify-content-between mb-3">
<div class=col-6>
<h1>Search</h1>
</div>
<div class="col-6 col-search-close">
<a class=js-search href=# aria-label=Close><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a>
</div>
</div>
<div id=search-box>
<input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...>
</div>
</section>
<section class=section-search-results>
<div id=search-hits>
</div>
</section>
</div>
</aside>
<div class=page-header>
<header class=header--fixed>
<nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main>
<div class=container-xl>
<div class="d-none d-lg-inline-flex">
<a class=navbar-brand href=../../><img src=../../media/logo_hu755fd628a59dfbfb6631c31ab0c8a3a3_17790_0x70_resize_lanczos_3.png alt="ERC DECODE"></a>
</div>
<button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar-content aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span>
</button>
<div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
<a class=navbar-brand href=../../><img src=../../media/logo_hu755fd628a59dfbfb6631c31ab0c8a3a3_17790_0x70_resize_lanczos_3.png alt="ERC DECODE"></a>
</div>
<div class="navbar-collapse main-menu-item collapse justify-content-end" id=navbar-content>
<ul class="navbar-nav d-md-inline-flex">
<li class=nav-item>
<a class=nav-link href=../../#DT_blank_about><span>About</span></a>
</li>
<li class=nav-item>
<a class=nav-link href=../../#DT_pages_news><span>News</span></a>
</li>
<li class=nav-item>
<a class=nav-link href=../../#DT_people_PIs><span>Team</span></a>
</li>
<li class=nav-item>
<a class=nav-link href=../../#DT_pages_publications><span>Research Output</span></a>
</li>
<li class=nav-item>
<a class=nav-link href=../../contact><span>Contact</span></a>
</li>
</ul>
</div>
<ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">
<li class=nav-item>
<a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a>
</li>
<li class="nav-item dropdown theme-dropdown">
<a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences">
<i class="fas fa-moon" aria-hidden=true></i>
</a>
<div class=dropdown-menu>
<a href=# class="dropdown-item js-set-theme-light">
<span>Light</span>
</a>
<a href=# class="dropdown-item js-set-theme-dark">
<span>Dark</span>
</a>
<a href=# class="dropdown-item js-set-theme-auto">
<span>Automatic</span>
</a>
</div>
</li>
</ul>
</div>
</nav>
</header>
</div>
<div class=page-body>
<div class=pub>
<div class="article-container pt-3">
<h1>Transcriptional landscape of rice roots at the single-cell resolution</h1>
<div class=article-metadata>
<div>
<span>
<a href=../../author/qing-liu/>Qing Liu</a></span>, <span>
<a href=../../author/zhe-liang/>Zhe Liang</a></span>, <span>
<a href=../../author/dan-feng/>Dan Feng</a></span>, <span>
<a href=../../author/sanjie-jiang/>Sanjie Jiang</a></span>, <span>
<a href=../../author/yifan-wang/>Yifan Wang</a></span>, <span>
<a href=../../author/zhuoying-du/>Zhuoying Du</a></span>, <span>
<a href=../../author/ruoxi-li/>Ruoxi Li</a></span>, <span>
<a href=../../author/guihua-hu/>Guihua Hu</a></span>, <span>
<a href=../../author/pingxian-zhang/>Pingxian Zhang</a></span>, <span>
<a href=../../author/yanfei-ma/>Yanfei Ma</a></span>, <span class=author-highlighted>
<a href=../../author/jan-lohmann/>Jan Lohmann</a></span>, <span>
<a href=../../author/xiaofeng-gu/>Xiaofeng Gu</a></span>
</div>
<span class=article-date>
March 2021
</span>
</div>
<div class="btn-links mb-3">
<a class="btn btn-outline-primary btn-page-header" href="https://reader.elsevier.com/reader/sd/pii/S1674205220304457?token=F88C07115365583305100B1463A5F82A1072D6103FF3B3AF2520D5458306E9957F6487A64E8BB6F6AFE145E946BC504C&originRegion=eu-west-1&originCreation=20230309160458" target=_blank rel=noopener>
PDF
</a>
<a class="btn btn-outline-primary btn-page-header" href=https://doi.org/10.1016/j.molp.2020.12.014 target=_blank rel=noopener>
DOI
</a>
</div>
</div>
<div class="article-header article-container featured-image-wrapper mt-4 mb-4" style=max-width:720px;max-height:851px>
<div style=position:relative>
<img src=../../publication/liu-2021/featured_hude1345dc9ebcd748d71be8c39ba67a44_3315369_720x2500_fit_q100_h2_lanczos.webp width=720 height=851 alt class=featured-image>
</div>
</div>
<div class=article-container>
<h3>Abstract</h3>
<p class=pub-abstract>There are two main types of root systems in flowering plants, namely taproot systems of dicots and fibrous root systems found in monocots. Despite this fundamental split, our current knowledge of cellular and molecular mechanism driving root development is mainly based on studies of the dicot model Arabidopsis. However, the world major crops are monocots and little is known about the transcriptional programs underlying cell-type specification in this clade. Here, we report the transcriptomes of more than 20 000 single cells derived from root tips of two agronomically important rice cultivars. Using combined computational and experimental analyses we were able to robustly identify most of the major cell types and define novel cell-type-specific marker genes for both cultivars. Importantly, we found divergent cell types associated with specific regulatory programs, including phytohormone biosynthesis, signaling, and response, which were well conserved between the two rice cultivars. In addition, we detected substantial differences between the cell-type transcript profiles of Arabidopsis and rice. These species-specific features emphasize the importance of analyzing tissues across diverse model species, including rice. Taken together, our study provides insight into the transcriptomic landscape of major cell types of rice root tip at single-cell resolution and opens new avenues to study cell-type specification, function, and evolution in plants.</p>
<div class=row>
<div class=col-md-1></div>
<div class=col-md-10>
<div class=row>
<div class="col-12 col-md-3 pub-row-heading">Type</div>
<div class="col-12 col-md-9">
<a href=../../publication/#2>
Journal article
</a>
</div>
</div>
</div>
<div class=col-md-1></div>
</div>
<div class="d-md-none space-below"></div>
<div class=row>
<div class=col-md-1></div>
<div class=col-md-10>
<div class=row>
<div class="col-12 col-md-3 pub-row-heading">Publication</div>
<div class="col-12 col-md-9"><em>Molecular Plant</em></div>
</div>
</div>
<div class=col-md-1></div>
</div>
<div class="d-md-none space-below"></div>
<div class=space-below></div>
<div class=article-style></div>
<div class=article-tags>
<a class="badge badge-light" href=../../tag/lohmann-group/>Lohmann Group</a>
</div>
<div class=share-box>
<ul class=share>
<li>
<a href="https://twitter.com/intent/tweet?url=https://erc-decode.github.io/decode-website-academic/publication/liu-2021/&text=Transcriptional%20landscape%20of%20rice%20roots%20at%20the%20single-cell%20resolution" target=_blank rel=noopener class=share-btn-twitter aria-label=twitter>
<i class="fab fa-twitter"></i>
</a>
</li>
<li>
<a href="https://www.facebook.com/sharer.php?u=https://erc-decode.github.io/decode-website-academic/publication/liu-2021/&t=Transcriptional%20landscape%20of%20rice%20roots%20at%20the%20single-cell%20resolution" target=_blank rel=noopener class=share-btn-facebook aria-label=facebook>
<i class="fab fa-facebook"></i>
</a>
</li>
<li>
<a href="mailto:?subject=Transcriptional%20landscape%20of%20rice%20roots%20at%20the%20single-cell%20resolution&body=https://erc-decode.github.io/decode-website-academic/publication/liu-2021/" target=_blank rel=noopener class=share-btn-email aria-label=envelope>
<i class="fas fa-envelope"></i>
</a>
</li>
<li>
<a href="https://www.linkedin.com/shareArticle?url=https://erc-decode.github.io/decode-website-academic/publication/liu-2021/&title=Transcriptional%20landscape%20of%20rice%20roots%20at%20the%20single-cell%20resolution" target=_blank rel=noopener class=share-btn-linkedin aria-label=linkedin-in>
<i class="fab fa-linkedin-in"></i>
</a>
</li>
<li>
<a href="whatsapp://send?text=Transcriptional%20landscape%20of%20rice%20roots%20at%20the%20single-cell%20resolution%20https://erc-decode.github.io/decode-website-academic/publication/liu-2021/" target=_blank rel=noopener class=share-btn-whatsapp aria-label=whatsapp>
<i class="fab fa-whatsapp"></i>
</a>
</li>
<li>
<a href="https://service.weibo.com/share/share.php?url=https://erc-decode.github.io/decode-website-academic/publication/liu-2021/&title=Transcriptional%20landscape%20of%20rice%20roots%20at%20the%20single-cell%20resolution" target=_blank rel=noopener class=share-btn-weibo aria-label=weibo>
<i class="fab fa-weibo"></i>
</a>
</li>
</ul>
</div>
<div class="media author-card content-widget-hr">
<a href=../../author/jan-lohmann/><img class="avatar mr-3 avatar-circle" src=../../author/jan-lohmann/avatar_hue25ada15bfb34099853d0cfcde0092bd_21139_270x270_fill_q100_lanczos_center.jpg alt="Jan Lohmann"></a>
<div class=media-body>
<h5 class=card-title><a href=../../author/jan-lohmann/>Jan Lohmann</a></h5>
<h6 class=card-subtitle>Principal Investigator</h6>
<ul class=network-icon aria-hidden=true>
<li>
<a href=../../contact/#DT_contact_lohmann>
<i class="fas fa-envelope"></i>
</a>
</li>
<li>
<a href=https://twitter.com/meristemania target=_blank rel=noopener>
<i class="fab fa-twitter"></i>
</a>
</li>
<li>
<a href="https://scholar.google.com/citations?user=BkbnIxkAAAAJ" target=_blank rel=noopener>
<i class="fas fa-graduation-cap"></i>
</a>
</li>
</ul>
</div>
</div>
<div class="article-widget content-widget-hr">
<h3>Related</h3>
<ul>
<li><a href=../../publication/feng-2022/>Chromatin accessibility illuminates single-cell regulatory dynamics of rice root tips</a></li>
</ul>
</div>
</div>
</div>
</div>
<div class=page-footer>
<div class=container>
<footer class=site-footer>
<p class=powered-by>
<a href=../../privacy/>Privacy Policy</a>
</p>
<p class=powered-by>
© Copyright, 2022 Heidelberg, EMBL, DKFZ, COS. This work was performed on behalf of the DECODE Consortium and funded under the European Research Council Synergy Grant DECODE under grant agreement No. 810296.
</p>
<p class=powered-by>
Published with <a href="https://wowchemy.com/?utm_campaign=poweredby" target=_blank rel=noopener>Wowchemy</a> — the free, <a href=https://github.com/wowchemy/wowchemy-hugo-themes target=_blank rel=noopener>open source</a> website builder that empowers creators.
</p>
</footer>
</div>
</div>
<div id=modal class="modal fade" role=dialog>
<div class=modal-dialog>
<div class=modal-content>
<div class=modal-header>
<h5 class=modal-title>Cite</h5>
<button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span>
</button>
</div>
<div class=modal-body>
<pre><code class="tex hljs"></code></pre>
</div>
<div class=modal-footer>
<a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank>
<i class="fas fa-copy"></i> Copy
</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank>
<i class="fas fa-download"></i> Download
</a>
<div id=modal-error></div>
</div>
</div>
</div>
</div>
<script src=../../js/vendor-bundle.min.b73dfaac3b6499dc997741748a7c3fe2.js></script>
<script src=https://cdn.jsdelivr.net/gh/desandro/imagesloaded@v4.1.4/imagesloaded.pkgd.min.js integrity="sha512-S5PZ9GxJZO16tT9r3WJp/Safn31eu8uWrzglMahDT4dsmgqWonRY9grk3j+3tfuPr9WJNsfooOR7Gi7HL5W2jw==" crossorigin=anonymous></script>
<script src=https://cdn.jsdelivr.net/gh/metafizzy/isotope@v3.0.6/dist/isotope.pkgd.min.js integrity="sha512-Zq2BOxyhvnRFXu0+WE6ojpZLOU2jdnqbrM1hmVdGzyeCa1DgM3X5Q4A/Is9xA1IkbUeDd7755dNNI/PzSf2Pew==" crossorigin=anonymous></script>
<script src=https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.2.1/build/highlight.min.js integrity="sha512-Ypjm0o7jOxAd4hpdoppSEN0TQOC19UtPAqD+4s5AlXmUvbmmS/YMxYqAqarQYyxTnB6/rqip9qcxlNB/3U9Wdg==" crossorigin=anonymous></script>
<script src=https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.2.1/build/languages/r.min.js crossorigin=anonymous></script>
<script src=https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.2.1/build/languages/python.min.js crossorigin=anonymous></script>
<script src=https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.2.1/build/languages/latex.min.js crossorigin=anonymous></script>
<script src=https://cdn.jsdelivr.net/npm/leaflet@1.7.1/dist/leaflet.min.js integrity crossorigin=anonymous></script>
<script id=search-hit-fuse-template type=text/x-template>
        <div class="search-hit" id="summary-{{key}}">
          <div class="search-hit-content">
            <div class="search-hit-name">
              <a href="{{relpermalink}}">{{title}}</a>
              <div class="article-metadata search-hit-type">{{type}}</div>
              <p class="search-hit-description">{{snippet}}</p>
            </div>
          </div>
        </div>
      </script>
<script src=https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin=anonymous></script>
<script src=https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin=anonymous></script>
<script id=page-data type=application/json>{"use_headroom":true}</script>
<script src=../../js/wowchemy-headroom.1cb9e2fc8399acee94eab837265b73bf.js type=module></script>
<script src=../../en/js/wowchemy.min.247fd8f54253895301106e3006f53f38.js></script>
<script async defer src=https://buttons.github.io/buttons.js></script>
</body>
</html>